pkgs/servers/dns/bind/default.nix

```diff
@@ -24,6 +27,10 @@ stdenv.mkDerivation rec {
 
   STD_CDEFINES = [ "-DDIG_SIGCHASE=1" ]; # support +sigchase
 
+  BUILD_CC = "cc";
```

Comment:
This also surprises me, I think the setup hook should also do this.

Reply:
https://gitlab.isc.org/isc-projects/bind9/blob/master/configure.in#L5125-5127

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/cc-wrapper/setup-hook.sh#L145